vllm/worker/model_runner.py

```diff
@@ -214,6 +215,8 @@ def __init__(
             input_positions: Optional[List[List[int]]] = None,
             mrope_input_positions: Optional[List[List[List[int]]]] = None,
 
+            # The number of original input tokens of each sequence
+            num_orig_input_tokens_list: Optional[List[List[int]]] = None,
```

Comment:
Can you explain / comment why this is a List of Lists? I understand the outer list is the sequence id, but what is the inner list for?

The other model runners are a single List, which is more intuitive to me

Reply:
@robertgshaw2-neuralmagic the goal was to ensure that `num_orig_input_tokens_list` has the same shape as `input_positions`. This would simplify switching the logic based on `num_orig_input_tokens_tensor` instead of the `positions` tensor within the `Phi3LongRoPEScaledRotaryEmbedding` class.